examples/openshift/README.md

```diff
@@ -20,8 +22,6 @@ Obtain you ingress subdomain:
 echo "apps.$(kubectl get dns cluster -o jsonpath='{ .spec.baseDomain }')"
 ```
 
-And update the examples/openshift/openshift-values.yaml file accordingly, replacing the APP_SUBDOMAIN value.
-
```

Comment:
Users shouldn't be editing most of the values files. they are from us to them.

The the example-your-values.yaml is for them to edit.